.github/workflows/continuous-integration.yml

```diff
@@ -92,7 +92,7 @@ jobs:
       - name: find broken wikilinks
         run: |
           ARTICLES=$( git diff --diff-filter=d --name-only ${{ github.sha }}^ ${{ github.sha }} "wiki/**/*.md" "news/*.md" )
-          [[ -n "$ARTICLES" ]] && python .github/scripts/ci/find_broken_wikilinks.py --target $ARTICLES
+          python .github/scripts/ci/find_broken_wikilinks.py --target $ARTICLES
```

Comment:
also we're lucky the articles don't have spaces in file names or this would break

Reply:
I copied it from your outdated check

Reply:
yeah I figured